### ISSUE TITLE
 Move from NcMultiselect to NcSelect

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/deck#4672]
NcMultiselect has lots of problems in regards to accessibility so at some point all usages should be switched to the new NcSelect component.

**Comment by juliushaertl** [comment]
Done with https://github.com/nextcloud/deck/pull/5621
